content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -204,50 +202,69 @@ Use `kubectl edit` to modify the size of a Cassandra StatefulSet.
       replicas: 3
     ```
 
-1. Change the number of replicas to 4, and then save the manifest. 
+1. Change the number of replicas to 4, and then save the manifest.
 
-    The `StatefulSet` now contains 4 Pods.
+    The StatefulSet now scales to run with 4 Pods.
 
-1. Get the Cassandra StatefulSet to verify:
+1. Get the Cassandra StatefulSet to verify your change:
 
     ```shell
     kubectl get statefulset cassandra
     ```
 
-    The response should be
+    The response should be similar to:
 
     ```
     NAME        DESIRED   CURRENT   AGE
     cassandra   4         4         36m
     ```
-      
+
 {{% /capture %}}
 
 {{% capture cleanup %}}
-Deleting or scaling a StatefulSet down does not delete the volumes associated with the StatefulSet. This setting is for your safety because your data is more valuable than automatically purging all related StatefulSet resources. 
+Deleting or scaling a StatefulSet down does not delete the volumes associated with the StatefulSet. This setting is for your safety because your data is more valuable than automatically purging all related StatefulSet resources.
 
 {{< warning >}}
 Depending on the storage class and reclaim policy, deleting the *PersistentVolumeClaims* may cause the associated volumes to also be deleted. Never assume you’ll be able to access data if its volume claims are deleted.
 {{< /warning >}}
 
-1. Run the following commands (chained together into a single command) to delete everything in the Cassandra `StatefulSet`:
+1. Run the following commands (chained together into a single command) to delete everything in the Cassandra StatefulSet:
 
     ```shell
-    grace=$(kubectl get po cassandra-0 -o=jsonpath='{.spec.terminationGracePeriodSeconds}') \
+    grace=$(kubectl get pod cassandra-0 -o=jsonpath='{.spec.terminationGracePeriodSeconds}') \
       && kubectl delete statefulset -l app=cassandra \
-      && echo "Sleeping $grace" \
+      && echo "Sleeping ${grace} seconds" 1>&2 \
       && sleep $grace \
-      && kubectl delete pvc -l app=cassandra
+      && kubectl delete persistentvolumeclaim -l app=cassandra
     ```
 
-1. Run the following command to delete the Cassandra Service.
+1. Run the following command to delete the Service you set up for Cassandra:
 
     ```shell
     kubectl delete service -l app=cassandra
     ```
 
 {{% /capture %}}
 
+{{% capture discussion %}}
+## Cassandra container environment variables
```

Comment:
@sftim , I think this information is important, but it does not show up on the preview.
Is there a `discussion` capture in the tutorial template? I only found this directive used in tasks?

Reply:
Good spot.